src/Constraints/BbanValidator.php

```diff
@@ -2,24 +2,11 @@
 
 namespace SLLH\IsoCodesValidator\Constraints;
 
-use IsoCodes;
-use SLLH\IsoCodesValidator\AbstractIsoCodesConstraintValidator;
-use Symfony\Component\Validator\Constraint;
+trigger_error('The '.__NAMESPACE__.'\BbanValidator class is deprecated since 1.2 and will be removed in 2.0. User '.__NAMESPACE__.'\IsoCodesGenericValidator instead', E_USER_DEPRECATED);
```

Comment:
I suggest following the Symfony standard here: trigger them silenced so that it does not impact normal error handlers. Symfony will still report them in the web debug toolbar.

and there is a typo: ``User`` vs ``Use``